core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -204,6 +204,12 @@ object TopicCommand extends Logging {
 
   def describeTopic(zkClient: KafkaZkClient, opts: TopicCommandOptions) {
     val topics = getTopics(zkClient, opts)
+    val ifExists = opts.options.has(opts.topicOpt)
```

Comment:
Since we have the same logic in three places now, maybe we may as well add a helper? Something like `ensureTopicExists`?

Reply:
yeah sure, actually we are not supporting if-exists option in describe command so moved common code to utils

Reply:
Moved common code to ensureTopicExists method, @hachikuji  @omkreddy Please take a look at this PR when you get a chance.